web/src/mostRecentBuild.ts

```diff
@@ -0,0 +1,69 @@
+import { Build } from "./types"
+import { isZeroTime } from "./time"
+
+export type ResourceWithBuilds = {
+  name: string
+  buildHistory: Array<Build>
+  pendingBuildSince: string
+  pendingBuildEdits: Array<string> | null
+}
+
+const buildByDate = (b1: BuildTuple, b2: BuildTuple) => {
+  let b1Date = Date.parse(b1.since)
+  let b2Date = Date.parse(b2.since)
+  if (b1Date > b2Date) {
+    return -1
+  }
+  if (b1Date < b2Date) {
+    return 1
+  }
+  return 0
+}
+
+type BuildTuple = {
```

Comment:
I'm not wild about the name of this type. Open to suggestions but it's not super urgent.